gulpfile.js

```diff
@@ -29,15 +28,28 @@ plugins.source = require('vinyl-source-stream');
 plugins.buffer = require('vinyl-buffer');
 plugins.pngquant = require('imagemin-pngquant');
 plugins.babelify = require('babelify');
-plugins.uglifyify = require('uglifyify');
 plugins.browserSync = require('browser-sync');
+plugins.interceptErrors = function (error) {
+    var args = Array.prototype.slice.call(arguments);
+
+    // Send error to notification center with gulp-notify
+    plugins.notify.onError({
+        title: 'Compile Error',
+        message: '<%= error.message %>'
+    }).apply(this, args);
```

Comment:
niet beter in console doen? Ik zet mijn notifications meestal af. En notification ziet ge ook maar effe.